### ISSUE TITLE
change UnetLoss function

### ISSUE THREAD

**Issue by joaocarvalho00** [joaocarvalho00/AutomatedMarineGrowth-Classification#16]
def UnetLoss(preds, targets):
    """
    Loss function for UNet
    """
    loss_function = nn.CrossEntropyLoss()
    ce_loss = loss_function(preds, targets)
    acc = (torch.max(preds, 1)[1] == targets).float().mean()
    return ce_loss, acc

no need to call nn.CrossEntropy every time,

build more general function
